src/components/GenderSelector.tsx

```diff
@@ -0,0 +1,91 @@
+import React, {useState} from 'react';
+import classes from './GenderSelector.module.scss';
+import '../index.scss';
+
+const GenderSelector = (props) => {
+    const [radio, setRadio] = useState('');
+    const [radioError, setRadioError] = useState(''); 
+
+    const blurHandler = () => {
+        if (radio === "") {
+            setRadioError('Please select Your gender');
+        } else {
+            setRadioError(''); 
+        }
+    }
+
+    return (
+        <div className={classes.gender__wrapper}>
+            <label className={classes.label}> 
+                <div className={classes.radio__wrapper}>
+                    <p className={classes.radio__title}>
+                        Select Your Gender
+                    </p>
+                    <div className={classes.radio__items}>
+                        <div className={classes.radio__item}> 
+                            <input 
+                                className={classes.radio__inner}
+                                type='radio'
+                                name='Male'
+                                checked={radio === 'Male'}
+                                onChange={e => {
+                                    setRadio(e.target.name);
+                                    setRadioError('');
+                                }}
+                                {...props} 
+                            />
+                            <p className={classes.radio__text}>
+                                Male
+                            </p>
+                        </div>
+                        <div className={classes.radio__item}> 
+                            <input 
+                                className={classes.radio__inner}
+                                type='radio'
+                                name='Female'
+                                checked={radio === 'Female'}
+                                onChange={e => {
+                                    setRadio(e.target.name);
+                                    setRadioError('');
+                                }}
+                                {...props} 
+                            />
+                            <p className={classes.radio__text}>
+                                Female
+                            </p>
+                        </div>
+                        <div className={classes.radio__item}> 
+                            <input 
+                                className={classes.radio__inner}
+                                type='radio'
+                                name='Other'
+                                checked={radio === 'Other'}
+                                onChange={e => {
+                                    setRadio(e.target.name);
```

Comment:
please, create constants for each selected types and use at the name, checked and here, in setRadio

const GENDER__MALE = 'male'


                            <input 
                                className={classes.radio__inner}
                                type='radio'
                                name={GENDER__MALE}
                                checked={radio === GENDER__MALE}
                                onChange={e => {
                                    setRadio(GENDER__MALE);
                                    setRadioError('');
                                }}
                                {...props} 
                            />

Reply:
done